### DISCUSSION TITLE
To reproduce exact transforms using applied_operations in metatensor

### DISCUSSION THREAD

**Discussion by Joeycho** [Project-MONAI/MONAI#5676] [Q&A] [answered]
Hello all,

For the 3D medical image segmentation task, I have a patch image (160,96,160) and a low-resolution upscaled label (720,440,720) from the same original image/label (720,440,720).

Is there a way to replicate the applied transforms of this patch to this low-resolution upscaled label? So, I can get a predicted low-resolution patch for this patch image. 

I can find there is a way to invert applied transforms, but I wonder how I can re-apply the exact transforms to others.

**Comment by KumoLiu** [comment, marked as answer]
Hi @Joeycho, for images, the reverse operation does not bring back the missing information, but only ensures that the shape is restored to the same. So if you want low-resolution data, maybe you can try to use [spacing](https://github.com/Project-MONAI/MONAI/blob/3873d236a732b31fbf33ed297bccaa1c2a93873f/monai/transforms/spatial/array.py#L423)
Hope it can help you, Thanks!

**Reply by Joeycho** [reply]
Ah, I see. There are some transforms, such as rotation, zooming, and they are applied with certain probability.

I just separate training, one  for low-resolution data, and use the result and image data for the next training, which enables cropping low-resolution prediction together with original images and labels.
